bioinf-tools-1/add_modules/input_output_module.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+
+def write_as_dict(input_fastq: str) -> dict:
+    seqs = {}
+    with open(input_fastq) as file:
+        for line in file:
+            if line.startswith("@"):
+                line1 = line.strip()
+                name = line1.split(" ")[0]
+                if not name:
+                    break
+                line2 = file.readline()
+                sequence = line2.strip()
+                line = file.readline()
+                line3 = file.readline()
+                quality = line3.strip()
+                seqs[name] = (sequence, quality)
+    return seqs
```

Comment:
```suggestion
def write_as_dict(input_fastq: str) -> dict:
    seqs = {}
    with open(input_fastq) as file:
        while True:
            line1 = file.readline()
            if not line1:
                break  # Достигли конца файла
            if line1.startswith("@"):
                name = line1.strip().split(" ")[0]
                sequence = file.readline().strip()
                file.readline()  # Пропускаем строку с '+'
                quality = file.readline().strip()
                seqs[name] = (sequence, quality)
    return seqs
```

Так будет покороче и попонятнее, а также не будет перезаписи похожих друг на друга переменных `line`